src/ptvsd/wrapper.py

```diff
@@ -1291,6 +1291,10 @@ def on_pydevd_thread_resume_single_notification(self, seq, args):
             # https://github.com/microsoft/ptvsd/issues/1530.
             self.send_event('continued', **body)
 
+    @pydevd_events.handler(pydevd_comm.CMD_EXIT)
+    def on_pydevd_cmd_exit(self, seq, args):
```

Comment:
@int19h do we need to specifically handle this case in the new adapter?

Reply:
I'm not entirely sure what this even is. The handler does nothing, so it's here because this notification gets received in that scenario, and it wasn't received previously. 

In any case, we do handle DAP "exited", and we do allow unknown events (we log them as errors, but continue running), so nothing should break - and if anything is missing, we'll find out quickly enough.

Reply:
This may be sent from the pydevd backend when it's about to exit... 

I'm reviewing the code and it's added as an `atexit` handler to add that message, but it kills all the threads right after adding that message, so, in practice it's only sent very seldomly (I added it  because a test failed due to that).

Thinking about it a bit more, I think what I did is not correct, this message should not even be sent when using the DAP backend. I've just provided a fix for this.